Cargo.toml

```diff
@@ -1,26 +1,31 @@
 [workspace]
 members = [
-  "crates/aesgcm",
-  "crates/anonymization",
-  "crates/cloudproof",
-  "crates/cover_crypt",
-  "crates/ecies",
-  "crates/ffi_utils",
-  "crates/findex",
-  "crates/fpe",
+    "crates/aesgcm",
+    "crates/anonymization",
+    "crates/cloudproof",
+    "crates/cover_crypt",
+    "crates/ecies",
+    "crates/ffi_utils",
+    "crates/findex",
+    "crates/fpe",
 ]
 # some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`. Fix the resolver value to avoid surprises.
 resolver = "1"
 
 [workspace.dependencies]
 async-trait = "0.1.51"
 base64 = "0.21.0"
-cosmian_crypto_core = "9.0.0"
-cloudproof_cover_crypt = "12.0.1"
+cosmian_crypto_core = "9.0"
+cloudproof_cover_crypt = "12.0"
```

Comment:
I thought we settled on using the full semver number since it prevents from using earlier patch releases while allowing to upgrad to the next patch/minor release.

Why did you remove the patch number ?